README.md

```diff
@@ -0,0 +1,57 @@
+# Scverse Ecosystem Packages
+
+This repository contains a list of approved software that works together with
+scverse core packages and are listed on our website. The goal is to increase
+visibility of ecosystem packages and make it easier for users to find
+appropriate software. In the future, we may also test releases of core packages against
+the test suites of ecosystem packages.
```

Comment:
Only the first sentence changed. Just made each sentence it's own line for ease of editing.

```suggestion
This repository contains the list of scverse ecosystem packages that are displayed on scverse.org.
The goal is to increase visibility of ecosystem packages and make it easier for users to find appropriate software. 
In the future, we may also test releases of core packages against the test suites of ecosystem packages.
```